discourse-doctor.rb

```diff
@@ -1,18 +1,26 @@
 #!/usr/bin/env ruby
+# coding: utf-8
 
 Dir.chdir("/") do
   require 'bundler/inline'
   require 'net/http'
   require 'net/smtp'
+  require 'pathname'
 
   gemfile(true, ui: false) do
     source 'https://rubygems.org'
 
     gem 'dnsruby'
     gem 'paint'
+    gem 'sys-filesystem' # for checking available disk space
```

Comment:
add `-b` for block size you are right (we should probably do this on discourse_docker too @ZogStriP ?) 

>      -b      Use (the default) 512-byte blocks.  This is only useful as a way to override an BLOCKSIZE specification
>              from the environment.

Do you have any evidence the two first wouldn’t be the case ?

And I do disagree on the last statement, having an external dependency with an API almost no one knows is not better than using unix tools in my book.